.github/workflows/installation.yml

```diff
@@ -0,0 +1,110 @@
+name: Installation
+on: [push]
+
+jobs:
+  linux-vm:
+    name: ${{ matrix.python-version }} - ${{ matrix.method }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - '2.7'
+          - '3.6'
+          - '3.7'
+        method:
+          - 'python ./install.py'
```

Comment:
We technically already test this type of install quite a lot for all platforms and all python versions. I think it would make sense to test this here, but only if we test `install.py` more in depth, like make sure it installs the rez scripts in the `rez` folder correctly, and on all platforms. We should also test https://github.com/nerdvegas/rez/blob/master/install.py#L81.

Without testing these things, I have the feeling we will just have jobs that will take CPU time without much benefits.

What do you think?

Reply:
I agree with testing `python -E`, let's do it in a separate PR. Originally these tests were written just to double check what is said in the [Installation Wiki page](https://github.com/nerdvegas/rez/wiki/Installation) actually holds true and is [not misleading beginners](https://github.com/nerdvegas/rez/pull/749).

So far, this is just a start and we can expand on it to more in-depth and [multi-platform tests](https://github.com/j0yu/bleeding-rez/runs/622011791?check_suite_focus=true) in the future to ensure whenever someone discovers this project and tries to follow the wiki step by step, it should _just work_.